### DISCUSSION TITLE
How to use this FreeRTOS library in PlatformIO

### DISCUSSION THREAD

**Discussion by 1988xd** [earlephilhower/arduino-pico#710] [Q&A] [answered]
I tried to add the FreeRTOS library to the PlatformIO project, but it didn't work, what should I do to make it run，thanks

**Comment by earlephilhower** [comment, marked as answer]
You can't use an external FreeRTOS library, you need to use the one included in the repo.  With that, you should be able to just `#include <FreeRTOS.h>` to enable it automatically, just like the example here.

**Reply by 1988xd** [reply]
Thank you for your answer，But I don't understand what is “repo” and how to use it.Can you tell me the detailed steps?

**Reply by Andy2No** [reply]
@1988xd "repo" is short for github repository.  I haven't tried Free RTOS, but you can read about using it with _this_ repo, in the libraries section, here: https://github.com/earlephilhower/arduino-pico/tree/master/libraries/FreeRTOS

Near the top of the page, you'll see a link to the example for it, as an Arduino sketch (.ino).
